### PR TITLE
add the basics of what the search works might look like

### PR DESCRIPTION
**Pull request by jamesgorrie**
Apologies - it's a little large, but what it has is:
* Search page with input
* Form uses [Formik](https://github.com/jaredpalmer/formik)
* When you submit, no page load!

__TODO (future PRs so we can make sure we're keeping track of what we're doing)__
* Loads of UI tidy
* Initial query to work
* If no initial query, don't get anything...

screen shot 2018-02-02 at 10 35 19

### CODE REVIEW

File: common/icons/clear.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const Clear = () => (
```
Comment (davidpmccormick) [inline common/icons/clear.js]: Just realised that we now have three variable name conventions on the go – ClassCase, camelCase and underscored. Given that icons are components and we want to be consistent, should we go for ClassCase everywhere?

File: common/icons/index.js [hunk 2]
```diff
@@ -16,5 +18,7 @@ export {
   search,
   zoomIn,
   zoomOut,
-  information
+  information,
```
Comment (davidpmccormick) [inline common/icons/index.js]: …and then do we capitalize these?

File: common/icons/clear.js (hunk 1 above)
Comment (jamesgorrie) [reply common/icons/clear.js]: Yip - my linter moaned at me, so I did this.

File: common/icons/index.js (hunk 2 above)
Comment (jamesgorrie) [reply common/icons/index.js]: Probably - next PR.

**Pull request merged**